performance_test/include/performance_test/executors.hpp

```diff
@@ -31,6 +31,7 @@ enum class ExecutorType
   SINGLE_THREADED_EXECUTOR = 1,
   STATIC_SINGLE_THREADED_EXECUTOR = 2,
   EVENTS_EXECUTOR = 3,
+  EVENTS_EXECUTOR_EXPERIMENTAL_NS = 4,
```

Comment:
I don;t think we should push both variants of the events-executor, one is sufficient

Reply:
Agree. I think the `humble` branch of the `ros2-performance` should be compatible with `humble` in `rclcpp`.
Since the experimental `EventsExecutor` is not present in `humble` of `rclcpp`, I'd remove the `EVENTS_EXECUTOR_EXPERIMENTAL_NS` here.

Reply:
I see the point, it makes sense to have a 1-to-1 match between the `humble` branches of both repos.

The one thing that caught my eye from these comments is that the changes in this PR I opened yesterday here: https://github.com/irobot-ros/rclcpp/pull/145
are bringing the experimental `EventsExecutor` to `irobot/humble`. So now I have these questions:
- Should those changes in `irobot-ros/rclcpp` be pointed towards `iron` instead of the `humble` branch?
- Otherwise, should we consider using the experimental `EventsExecutor` here?

Reply:
I'd leave only the executor from the `irobot_events_executor` since otherwise, this repo branch would only work with our own version of `rclcpp` (the one that has/will have the experimental/upstream version).
Maybe we can have a separate branch like `irobot/humble-with-upstream-events-executor` in which we use the upstream one.

Reply:
Note that the `irobot_events_executor` repo also requires a custom rclcpp branch as far as I remember